apps/launchpad/app/components/OnboardingModal/constants.ts

```diff
@@ -0,0 +1,56 @@
+export const STEPS = {
+  INTRO: 1,
+  TOPICS: 2,
+  SUBTOPICS: 3,
+  SUMMARY: 4,
+} as const
+
+export const MIN_SELECTIONS = 1
+export const MAX_SELECTIONS = 5
+
+export const INITIAL_TOPICS = [
```

Comment:
We're going to have to think about how this'll work for the query/mutation we end up using. Noting for later.